src/StackManagerVersion.h.in

```diff
@@ -12,3 +12,5 @@
 //
 
 const QString BUILD_VERSION = "@BUILD_SEQ@";
+const QString URL_BASE = "@BASE_URL@";
+const bool PR_BUILD = @PR_BUILD@;
```

Comment:
Just curious: why is this file called **StackManagerVersion.h.in** instead of **StackManagerVersion.h**?  What is the extra **.in** for?